src/Microsoft.AspNetCore.Sockets/HttpConnectionDispatcher.cs

```diff
@@ -378,6 +359,28 @@ private ConnectionState CreateConnection(HttpContext context)
             }
         }
 
+        private async Task<IList<Message>> ExtractMessages(HttpContext context, byte[] buffer)
```

Comment:
@BrennanConroy  - I will update ParseSendBatch to return `List<Message>` instead of `IList<Message>` because this is where IList is being returned initially from.